de_web_plugin.cpp

```diff
@@ -4301,17 +4305,34 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const deCONZ::
 
                 case TEMPERATURE_MEASUREMENT_CLUSTER_ID:
                 {
+
                     if (modelId == QLatin1String("VOC_Sensor"))
                     {
                         fpHumiditySensor.inClusters.push_back(ci->id());
                     }
-                    fpTemperatureSensor.inClusters.push_back(ci->id());
+                    
+                    // Hive devices, don't show model id faster enought
+                    if ((node->nodeDescriptor().manufacturerCode() == VENDOR_ALERTME) && (modelId.isEmpty()))
+                    {
+                    }
+                    // Don't create entry for the plug
+                    else if (modelId == QLatin1String("SLP2b"))
+                    {
+                    }
+                    // Don't create entry for cluster 0x07 and 0x08
+                    else if ((modelId == QLatin1String("SLR2")) && (i->endpoint() > 0x06 ))
+                    {
+                    }
+                    else 
+                    {
+                        fpTemperatureSensor.inClusters.push_back(ci->id());
+                    }
```

Comment:
Maybe better with `||` to safe some lines?

Reply:
IDK, It don't make a big difference on compiled file.
And I think like this it's easier to understand, It s not like your previous comment (the model enumeration)
The first test is to skip the test if there is no model ID, the second is a modelid check, the third is endpoint specific and can change later (there is some "black zone" on theses devices)